components/exchange/Translation.js

```diff
@@ -80,6 +85,11 @@ const TranslationContainer = styled.div`
   @media screen and (min-width: ${screenSizes.smallPhone.max}) {
     grid-template-columns: 2fr 3fr;
   }
+
+  @media screen and (max-width: ${screenSizes.smallPhone.max}) {
```

Comment:
If we mix min and max-width we can have problems in the boundary. At exactly `screenSizes.smallPhone.max`px some of the changes are active but not all. This can look very jarring when resizing the screen.

Reply:
Fixed however I did change it so that `WordClass` and `HeaderTitleType` change at `max-width` so that it looks nice at the boundary, otherwise it was still coming up with layout issues